skyportal/enum_types.py

```diff
@@ -74,6 +74,29 @@ def force_render_enum_markdown(values):
 
 LISTENER_CLASSES = [getattr(facility_apis, c) for c in LISTENER_CLASSNAMES]
 
+ANALYSIS_TYPES = ('lightcurve_fitting', 'spectrum_fitting')
+ANALYSIS_INPUT_TYPES = ('photometry', 'spectra', 'redshift', 'annotations', 'comments')
```

Comment:
Maybe meta analyses of the observations?

Reply:
Done.